CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 ## 6.9.0 (Unreleased)
 
+NOTES:
+* provider: minor refactoring of how we run acceptance tests in VCR mode ([#8425](https://github.com/hashicorp/terraform-provider-google-beta/pull/8425))
+
```

Comment:
We might want to remove this, but figured I'd flag for others to give their opinion on.

```suggestion
```